aspnetcore/security/cors.md

```diff
@@ -80,19 +103,23 @@ Enabling CORS on a per-endpoint basis using `RequireCors` currently does ***not*
 
 With endpoint routing, CORS can be enabled on a per-endpoint basis using the <xref:Microsoft.AspNetCore.Builder.CorsEndpointConventionBuilderExtensions.RequireCors*> set of extension methods:
 
-[!code-csharp[](cors/3.1sample/Cors/WebAPI/StartupEndPt.cs?name=snippet)]
+[!code-csharp[](cors/3.1sample/Cors/WebAPI/StartupEndPt.cs?name=snippet2&highlight=3,7-15,32,41,44)]
 
 In the preceding code:
 
-* `app.UseCors` enables the CORS middleware but doesn't add a policy. `app.UseCors` by itself, without a policy, does ***not*** allow cross-origin requests.
+* `app.UseCors` enables the CORS middleware with the default policy. Because no default policy has be specified, `app.UseCors()` doesn't enable CORS.
 * The `/echo` and controller endpoints allow cross-origin requests using the specified policy.
-* The `/echo2` and Razor Pages endpoints do ***not*** allow cross-origin requests.
+* The `/echo2` and Razor Pages endpoints do ***not*** allow cross-origin requests because no default policy was specified.
+
+The [[DisableCors]](#dc) attribute does ***not***  disable CORS that has been enabled by endpoint routing.
```

Comment:
I'm not sure how to improve this, but I don't know if it's clear that "endpoint routing" here means the call to `RequireCors`.

Reply:
The [[DisableCors]](#dc) attribute does ***not***  disable CORS that has been enabled by endpoint routing with `RequireCors`.